hack/multi-cluster-test.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright 2023 The Primaza Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+TAG=$(git rev-parse --short HEAD)
+
+CONTROLLER=primaza-controller:${TAG}
+AGENTAPP=agentapp:${TAG}
+AGENTSVC=agentsvc:${TAG}
+
+if [[ ! -e bin/yq ]]; then
+    make yq
+fi
+
+kind delete clusters worker controller
```

Comment:
Using `primaza` as a prefix for the clusters breaks this script, since this snippet in `test-acceptance-wip` deletes all clusters prefixed with `primaza` before running tests.

I've fixed this behavior in 0e84c39; these clusters are now removed only if we're using `kind` as a cluster provider.